### ISSUE TITLE
QUAST module to dynamically determine

### ISSUE THREAD

**Issue by schorlton** [ewels/MultiQC#1635]
### Description of feature

Currently, the QUAST module sets length modifiers and suffixes based on reasonable defaults: https://github.com/ewels/MultiQC/blob/d5866563e294e442025324eeb8e687c0e4752abe/multiqc/modules/quast/quast.py#L33-L40

This is great for the vast majority of people! In certain cases, however, this may not make sense as a user may not be able to either a) configure the QUAST module themselves or b) know _a priori_ the length of the thing they sequenced/assembled (eg. metagenomics)

Thereferore, I'm wondering if it's possible to dynamically set these values. Something like (pseudocode):
```python3
if median_length(assemblies) < 1000:
    suffix = "bp"
if 1000 <= median_length(assemblies) < 1000000:
    suffix = "kbp"
if median_length(assemblies) >= 1000000:
    suffix = "mbp"
```

Thanks for your consideration!

**Comment by ewels** [comment]
Yup, as long as the config value is used if set, then I think having a dynamic default sounds sensible 👍🏻
